docs/pages/blog/mui-x-v5.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Introducing MUI X v5.0
+description: We are excited to introduce MUI X v5.0.0!
+date: 2021-11-12T00:00:00.000Z
+authors:
+  ['oliviertassinari', 'm4theushw', 'flaviendelangle', 'DanailH', 'alexfauquette']
+card: true
+---
+
+We are excited to introduce [MUI X v5.0.0](https://github.com/mui-org/material-ui-x/releases/tag/v5.0.0)!
+
+<!-- TODO: upload a MUI X v5 specific banner -->
+<img src="/static/blog/mui-x-v5/card.png" alt="" style="width: 100%; margin-bottom: 16px;" />
+
+This release features some major highlights:
```

Comment:
Not sure "High-level goals" is a highlight :)

Reply:
I took the structure from here https://mui.com/blog/mui-core-v5/. Granted the `Thank you` is not on the list there. I will remove it. Not sure what to do for with rest.